### ISSUE TITLE
How to install quote typo

### ISSUE THREAD

**Issue by wmacevoy** [wxMaxima-developers/wxmaxima#1101]
In the "How to install" rtf document in Maxima-5.36.1, the last line on installing  gnuplot is incorrect:

```bash
echo 'set_plot_option([gnuplot_term, qt])$’ >> maxima-init.mac
```

The closing quote is not a single quote.   This will create an error that someone not familiar with the terminal will not know how to fix.

**Comment by wmacevoy** [comment]
The correct value for the location of maxima should be given as well.  Most typically:

/Applications/Maxima.app/Contents/Resources/maxima.sh

This is impossible to know for someone not familiar with the command line.

**Comment by gunterkoenigsmann** [comment]
I don't own a Mac so any help with the instructions is Welcome. Will Update tje instructions as soon as possible.

**Comment by wmacevoy** [comment]
The bundled version of gnuplot does not support "pngcairo" which is set to be be used by default in gnuplot.  So plotting fails until you uncheck "use cairo" in the preferences setting.  This took a bit to figure out.  It looks like the cairo is much nicer - probably the difference between useable for publication or not; I hope the bundled gnuplot would just support it.

**Comment by gunterkoenigsmann** [comment]
For publications I always write my own mydraw() command that first plots the scene using wxdraw and then generates a pdf file using draw(). Alternatively if wxMaxima is new enough you can right-click on the plot, open gnuplot's interactively and tell gnuplot's to save the drawing as SVG or similar.

**Comment by wmacevoy** [comment]
The .dmg version just gives a "sad X" with no useful options.  I created a pull request #1102 with updated installation documentation for OS X.  I do not think very many new OS X users get through this without the additional information.

**Comment by KubaO** [comment]
Is this still an issue? PR #1102 has been merged already.

**Comment by KubaO** [comment]
The RTF file has had this fixed in 75745aa8f, so this is now resolved.
